### ISSUE TITLE
Env Variables differ

### ISSUE THREAD

**Issue by danielfliegauf** [ManuelDeLeon/sveltekit-firebase-ssr#1]
Hello,

i have trouble with replicating because i think you are using Variables in your code, which are not setup:

VITE_FIREBASE_PRIVATE_KEY="Your key, including all \n"
VITE_FIREBASE_CLIENT_EMAIL="client email"
VITE_FIREBASE_PROJECT_ID="project id"
VITE_FIREBASE_CONFIG='Your Firebase config json, stringified'

and in the src you have something like: 

	FIREBASE_CLIENT_CONFIG = (import.meta.env.VITE_FIREBASE_CLIENT_CONFIG || '').toString();
	FIREBASE_SERVER_CONFIG = (import.meta.env.VITE_FIREBASE_SERVER_CONFIG || '').toString();

Maybe i am understanding something wrong here? 🤔 
Thank you :)
Daniel

**Comment by ManuelDeLeon** [comment]
Sorry about that. I had to change the config because Firebase wouldn't work on the server when deployed (even though it works locally on preview). I forgot to update the readme. See the new changes: https://github.com/ManuelDeLeon/sveltekit-firebase-ssr/commit/cd2b0126fcae74e51c9e3f1a9cb44f883a03d0fc

I can't believe someone's using it already (I just made it). Let me know how it goes with this starter project. I'm pretty happy with it so far. Here's a live version: https://sveltekit-firebase-ssr.vercel.app/
